api/turing/models/enricher.go

```diff
@@ -6,17 +6,19 @@ type Enricher struct {
 	Model
 	// Fully qualified docker image string used by the enricher, in the
 	// format registry/repository:version.
-	Image string `json:"image" validate:"required"`
```

Comment:
Cleaning up unused validate tags and the validation is done on the `EnricherEnsemblerConfig` struct declared in the request package instead of this.